jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/secondLevelCache/InfinispanCacheDeploymentListener.java

```diff
@@ -57,15 +62,8 @@
     public static final String CACHE_TYPE = "cachetype";    // shared (jpa) or private (for native applications)
     public static final String CACHE_PRIVATE = "private";
     public static final String CONTAINER = "container";
-    public static final String COLLECTION = "collection";
-    public static final String ENTITY = "entity";
-    public static final String IMMUTABLE_ENTITY = "immutable-entity";
     public static final String NAME = "name";
-    public static final String NATURAL_ID = "natural-id";
-    public static final String QUERY = "query";
-    public static final String TIMESTAMPS = "timestamps";
-    public static final String PENDING_PUTS = "pending-puts";
-    public static final String CUSTOM = "custom";
+    public static final String CACHES = "caches";
```

Comment:
Please remind me, where is the code that implements "caches"?  Thanks!

Reply:
https://github.com/wildfly/wildfly/pull/10639/files/ca191c84ccfe190219edb90824a602bb71a0dca2#diff-d5db9279d2617193a1ccc79e56a0fad7R95